frontstage/views/session.py

```diff
@@ -1,14 +1,31 @@
-from datetime import datetime, timezone
-
-from flask import Blueprint, jsonify, request
+from flask import Blueprint, jsonify, make_response, redirect, request, session, url_for
 
 from frontstage.common.authorisation import jwt_authorization
+from frontstage.common.session import Session
 
 session_bp = Blueprint("session", __name__)
 
 
-@session_bp.route("/expires-at", methods=["GET"])
+@session_bp.route("/expires-at", methods=["GET", "PATCH"])
```

Comment:
In this instance I think it is easier to split the methods out i.e have one for PATCH and one for GET. For the one that is looking after patch just use @jwt_authorization(request) rather than 
@jwt_authorization(request, refresh_session=False) and it will do the session extension for you

Reply:
I've updated the code based on the recommendations.